### DISCUSSION TITLE
TiddlyWiki in the wild

### DISCUSSION THREAD

**Discussion by dbohdan** [TiddlyWiki/TiddlyWiki5#7731] [General]
<https://tiddlywiki.com/static/Examples.html> could use more examples.

One that immediately comes to my mind is [_Singularity! A Tough Guide to the Rapture of the Nerds_](https://www.antipope.org/charlie/blog-static/fiction/accelerando/toughguide.html), a 2005 satirical nonfiction hypertext by science fiction writer Charles Stross. I think it is a good candidate for inclusion: a famous writer published a work directly as a TiddlyWiki. (Version 1.2.11!) I think it was many people's first exposure to TiddlyWiki, including mine.

**Comment by Jermolene** [comment]
Thank you @dbohdan for many years I used to show this as an example of TiddlyWiki. I knew it at its original URL of http://www.accelerando.org/_static/toughguide.html, and hadn't tracked it to its new home at antipope.org. I had assumed that it had been removed, and had been using an archive.org snapshot, but it's much nicer to be able to link to the authors own site.

I'll add a link soon.

**Comment by dbohdan** [comment]
Great! I am glad I could help.
